pkg/config/validation/validation.go

```diff
@@ -449,16 +449,55 @@ var ValidateDestinationRule = registerValidateFunc("ValidateDestinationRule",
 			errs = appendErrors(errs, validateSubset(subset))
 		}
 
-		errs = appendErrors(errs, validateExportTo(rule.ExportTo))
+		errs = appendErrors(errs, validateExportTo(namespace, rule.ExportTo, false))
 		return
 	})
 
-func validateExportTo(exportTo []string) (errs error) {
+func validateExportTo(namespace string, exportTo []string, isServiceEntry bool) (errs error) {
 	if len(exportTo) > 0 {
-		if len(exportTo) > 1 {
-			errs = appendErrors(errs, fmt.Errorf("exportTo should have only one entry (. or *) in the current release"))
-		} else {
-			errs = appendErrors(errs, visibility.Instance(exportTo[0]).Validate())
+		// Make sure there are no duplicates
+		exportToMap := make(map[string]struct{})
+		for _, e := range exportTo {
+			key := e
+			if visibility.Instance(e) == visibility.Private {
+				// substitute this with the current namespace so that we
+				// can check for duplicates like ., namespace
+				key = namespace
+			}
+			if _, exists := exportToMap[key]; exists {
+				if key != e {
+					errs = appendErrors(errs, fmt.Errorf("duplicate entries in exportTo: . and current namespace %s", namespace))
+				} else {
+					errs = appendErrors(errs, fmt.Errorf("duplicate entries in exportTo for entry %s", e))
+				}
+			} else {
+				// if this is a serviceEntry, allow ~ in exportTo
+				if isServiceEntry && visibility.Instance(e) == visibility.None {
```

Comment:
why make SE unique here?

Reply:
I am not sure if we will have a definitive use case for this but if we ever do, the only config that can support ~ is the serviceEntry - think of the scenario where you have a cronjob with some dummy service assigned to it, and you want to make sure no one can talk to the cronjob.

Reply:
Then the question comes, if it is not intended to be accessed, why even create a SE/service for it?

Reply:
For the proxies that implement that service - so that the GetProxyServiceInstances() doesn't return empty. Also, if a service is present, then metrics will be present. E.g., a cron job that calls a bunch of internal APIs every 10s..